packages/react/types/jsx-namespace.d.ts

```diff
@@ -25,8 +25,9 @@ export namespace EmotionJSX {
     extends ReactJSXElementAttributesProperty {}
   interface ElementChildrenAttribute extends ReactJSXElementChildrenAttribute {}
 
-  type LibraryManagedAttributes<C, P> = WithConditionalCSSProp<P> &
+  type LibraryManagedAttributes<C, P> = WithConditionalCSSProp<
```

Comment:
oh, this will clash with this: https://github.com/emotion-js/emotion/pull/1952/files#r463502611 . it has been mentioned that the distribution fixes that problem, might need to apply this here, will take a look later